src/Data/List/Relation/Binary/Sublist/Heterogeneous/Properties.agda

```diff
@@ -444,3 +447,192 @@ module _ {a e r} where
   decPoset ER-poset = record
     { isDecPartialOrder = isDecPartialOrder ER.isDecPartialOrder
     } where module ER = DecPoset ER-poset
+
+------------------------------------------------------------------------
+-- Properties of disjoint sublists
+
+module Disjointness {a b r} {A : Set a} {B : Set b} {R : REL A B r} where
+
+  private
+    infix 4 _⊆_
+    _⊆_ = Sublist R
+
+  -- Irreflexivity
+
+  Disjoint-irrefl : ∀{xs ys} {τ : xs ⊆ ys} → Disjoint τ τ → Null xs
```

Comment:
Hmm this is the opposite way round to how the standard library usually defines irreflexivity. Normally its "equals to" implies "not related". Obviously we don't have a notion of equivalence here, but I think it would still be more consistent to prove not disjoint.

Reply:
Note that _not Null implies not Disjoint_ is a trivial contraposition of my property.  My property seems strictly more useful than 
```agda
∀{x xs ys} {τ : (x :: xs) ⊆ ys} → Disjoint τ τ → \bot
```  
You get the latter as a special case of my property when instantiating `xs` to a non-empty list.